third_party/blake3/blake3.BUILD

```diff
@@ -30,7 +30,9 @@ cc_library(
     ] + select({
         "@bazel_tools//src/conditions:linux_x86_64": [
             "c/blake3_avx2_x86-64_unix.S",
-            "c/blake3_avx512_x86-64_unix.S",
+            # Disable to appease bazel-ci which uses ubuntu-18 (EOL) and GCC 7
+            # lacking the headers to compile AVX512.
+            # "c/blake3_avx512_x86-64_unix.S",
```

Comment:
@meteorcloudy FYI, this is an unfortunate consequence of the old versions running in CI.

Reply:
Related: https://github.com/bazelbuild/bazel/pull/18784#issuecomment-1609421453